app/src/main/kotlin/com/intive/picover/main/navigation/NavigationResult.kt

```diff
@@ -0,0 +1,28 @@
+package com.intive.picover.main.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.State
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.mutableStateMapOf
+import androidx.compose.runtime.remember
+import cafe.adriel.voyager.navigator.bottomSheet.BottomSheetNavigator
+import kotlin.reflect.KClass
+
+private val results = mutableStateMapOf<KClass<out Any>, Any?>()
```

Comment:
Custom result handling, until we get the official support from voyager https://github.com/adrielcafe/voyager/pull/128#issuecomment-1803298948